packages/website/src/bin/handle-config.js

```diff
@@ -43,7 +43,7 @@ const processConfig = (cwd, providedConfig = {}) => {
     const docsPath = path.resolve(cwd, docPath);
 
     const certainUrlPath = urlPath || path.relative(cwd, docsPath);
-    return { docsPath, name, description, urlPath: certainUrlPath };
+    return { docsPath, name, urlPath: certainUrlPath, ...docRest };
```

Comment:
docRest is spreading description and imgSrc here.

Reply:
In part it's that this is still `js` not `ts`, but this kind of arbitrary object spread feels scary to me.

Pretty sure ts-ing these files is out of scope for this, so maybe we just live with this for a bit?

Reply:
I'll remove the spread, and call out description/imgSrc directly here.